reference/docs-conceptual/samples/Repeating-a-Task-for-Multiple-Objects--ForEach-Object-.md

```diff
@@ -20,26 +26,36 @@ Size         : 203912880128
 VolumeName   : Local Disk
 ```
 
-We can convert the FreeSpace value to megabytes by dividing each value by 1024 twice; after the first division, the data is in kilobytes, and after the second division it is megabytes. You can do that in a ForEach-Object script block by typing:
+We can convert the **FreeSpace** value to megabytes by dividing each value by 1MB. You can do that
+in a `ForEach-Object` script block by typing:
 
+```powershell
+Get-CimInstance -Class Win32_LogicalDisk |
+  ForEach-Object -Process {($_.FreeSpace)/1MB}
 ```
-PS> Get-WmiObject -Class Win32_LogicalDisk | ForEach-Object -Process {($_.FreeSpace)/1024.0/1024.0}
+
+```Output
 48318.01171875
 ```
 
-Unfortunately, the output is now data with no associated label. Because WMI properties such as this are read-only, you cannot directly convert FreeSpace. If you type this:
+Unfortunately, the output is now data with no associated label. Because WMI properties such as this
+are read-only, you cannot directly convert **FreeSpace**. If you type this:
 
 ```powershell
-Get-WmiObject -Class Win32_LogicalDisk | ForEach-Object -Process {$_.FreeSpace = ($_.FreeSpace)/1024.0/1024.0}
+Get-CimInstance -Class Win32_LogicalDisk |
+  ForEach-Object -Process {$_.FreeSpace = ($_.FreeSpace)/1MB}
 ```
 
 You get an error message:
 
-```output
+```Output
 "FreeSpace" is a ReadOnly property.
-At line:1 char:70
-+ Get-WmiObject -Class Win32_LogicalDisk | ForEach-Object -Process {$_.F <<<< r
-eeSpace = ($_.FreeSpace)/1024.0/1024.0}
+At line:2 char:28
++   ForEach-Object -Process {$_.FreeSpace = ($_.FreeSpace)/1MB}
++                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    + CategoryInfo          : NotSpecified: (:) [], SetValueException
```

Comment:
This and the next line are overly indented. The +s from the error should line up.

Reply:
Fixed